src/assets/js/archive.js

```diff
@@ -5,62 +5,109 @@ const releasesToShow = 99999;
 function fetchFlutterReleases(os, callback, errorCallback) {
   // OS: windows, macos, linux
   const url = `https://storage.googleapis.com/flutter_infra_release/releases/releases_${os}.json`;
-  $.ajax({
-    type: 'GET',
-    url: url,
-    dataType: 'json',
-    success: function (data) { callback(data, os); },
-    error: function (xhr, textStatus, errorThrown) {
-      if (errorCallback) errorCallback(os);
-    }
-  })
+  fetch(url, { type: 'GET'})
+    .then(response => response.json())
+    .then(data => callback(data, os))
+    .catch(_ => { if (errorCallback) errorCallback(os) })
 }
 
 function updateTable(releases, os) {
   const releaseData = releases.releases;
 
   for (const channel in releases.current_release) {
-    const table = $(`#downloads-${os}-${channel}`);
-    table.addClass('collapsed').find('.loading').remove();
+    const table = document.getElementById(`downloads-${os}-${channel}`);
+
+    //table is not present when channel is dev
+    if(!table) {
+      continue;
+    }
 
-    const releasesForChannel = releaseData.filter(function (release) {
+    table.classList.add('collapsed');
+    const loadingElements = table.querySelectorAll('.loading');
+    loadingElements.forEach(function(element) {
+      element.remove();
+    });
+
+    const releasesForChannel = releaseData.filter(function(release) {
```

Comment:
This is a minor nit especially since we don't have a formatter setup for JS, but to better match our style for Dart code:

```suggestion
    const releasesForChannel = releaseData.filter(function (release) {
```